### DISCUSSION TITLE
Drip 300watts back to the grid

### DISCUSSION THREAD

**Discussion by visitdigital** [wills106/homeassistant-solax-modbus#489] [Q&A]
Hi, I think we have a Chint meter issue on our Solax battery, where 300W is continuously being pulled from the grid. The battery cannot see loads over about 3600watts either so something is definitely up with it. In the interim between getting it fixed is there a way i can fool the battery into pushing 300 watts back to the grid to offset it? You can see in the attached image there is ALWAYS at least 300watts coming from the grid. 
![image](https://github.com/wills106/homeassistant-solax-modbus/assets/1809171/57c6b505-02a6-4069-b5f0-36c8ae6a9bba)

**Comment by wills106** [comment]
What Inverter do you have?

If you have a Gen4 SolaX Hybrid you have the "Pgrid Bias" option #223

**Reply by visitdigital** [reply]
X1-AC
